src/tokenizer.rs

```diff
@@ -133,7 +133,7 @@ impl DefaultTokenizer {
                     quote_pairs.push(QuotePair {
                         start: start.pos,
                         end: current.pos,
-                        quotation: current.quotation,
+                        _quotation: current.quotation,
```

Comment:
Hm, does this mean we are not using `_quotation`? It has been so long since I've last touched this code so I do not know what my intent was with this field. Can we just remove it?

Reply:
It's unnecessary. This is basically used to turn lines into slices of normal or quoted blobs, and the quotation part is only necessary to create the pairs but not of the pairs themselves. So I removed it. 

If you are interested (this is optional), I captured some [investigation notes](https://github.com/deltatraced/delta-trace/blob/10f88eb2511d863cd42422cac1747cbb05a0f703/lan/tasks/weekly/2025/Wk%2037%20000%20Addressing%20shi%20PR%2010/tasks/000%20Looking%20into%20quotation%20removal.md) looking into it.